typescript/infra/config/environments/mainnet3/agent.ts

```diff
@@ -233,6 +226,32 @@ const metricAppContexts = [
     name: 'renzo_ezeth',
     matchingList: routerMatchingList(renzoEzEthAddresses),
   },
+  {
+    name: 'renzo_ezeth_old',
+    // There's an old message to Base that's stuck around, we
+    // just care about this one for now.
```

Comment:
do we need to keep this in the config forever or just until it gets delivered? wondering if it's worth just letting the RC cover this and not commiting this

Reply:
nah I was hoping to remove it after it's processed. Just wanted it in the hyperlane context so that we can always easily see that the one message is still stuck around